Sources/Fuzzilli/FuzzIL/AbstractInterpreter.swift

```diff
@@ -219,6 +219,32 @@ public struct AbstractInterpreter {
         setType(of: v, to: t)
     }
 
+    private func calleeTypes(for signature: FunctionSignature) -> [Type] {
+
+        func processType(_ type: Type) -> Type {
+            if type == .anything {
```

Comment:
I think this function should just be
```swift
if type == .anything {
  // .anything in the caller maps to .unknown in the callee
  return .unknown
}
return type
```
And we'll deal with the array type below, which should just be used for `.rest`